examples/extensions/update_sitelink_campaign_extension_setting.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Updates the sitelink campaign extension setting.
+
+Replaces the extension feed items of the sitelink campaign extension setting
+with the given feed item IDs. Note that this doesn't completely remove your
+old extension feed items. See https://developers.google.com/google-ads/api/docs/extensions/extension-settings/overview
```

Comment:
The url should go on a new line here. It'll probably exceed 80 chars anyway but we normally do this just to be as close to under 80 as possible.

Reply:
ack